docs/dashboards/dashboards-overview.asciidoc

```diff
@@ -4,6 +4,13 @@
 
 The following sections describe the {security-app}'s prebuilt dashboards, which provide visualizations of your security environment.
 
+You can also create and view custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard** in the upper-right corner. Once created, custom dashboards appear at the bottom of the page:
```

Comment:
```suggestion
You can also create and access custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard**. Once created, custom dashboards appear on the page:
```

Reply:
Made some slight modifications based on our guidelines for writing for accessibility which recommend avoiding directional language. 
https://docs.elastic.dev/content-design/accessibility

(I sometimes forget about these guidelines so I'm sure I'll be giving myself the same edits soon ;) )

Reply:
Accessibility is tricky. Tbh, sometimes I think it's necessary, especially in a complex or "noisy" UI, but understand why there are guidelines in there to suggest against directional language. You could also say something like `"Once created, confirm that your custom dashboard appears on the page." `